src/platform/intel/cavs/memory.c

```diff
@@ -74,7 +74,10 @@ static struct block_map rt_heap_map[] = {
 /* Heap blocks for buffers */
 static struct block_hdr buf_block[HEAP_BUFFER_COUNT];
 static struct block_hdr hp_buf_block[HEAP_HP_BUFFER_COUNT];
+/* To be removed if Apollolake gets LP memory*/
+#ifndef CONFIG_APOLLOLAKE
 static struct block_hdr lp_buf_block[HEAP_LP_BUFFER_COUNT];
+#endif
```

Comment:
Thanks for pointing it out. Do you know where we can get know whether LP memory is supported or not by HW?

Reply:
@lbetlej is a specialist in this field i got my information from him.